microsoftgraph/microsoft-graph-explorer-v4
# Feature: Display blue dot present adaptive cards

## Overview
Adds a blue dot on the queries that are adaptive cards supported.

Fixes #470 

### Demo

Present:
![image](https://user-images.githubusercontent.com/58787602/80593050-26f36900-8a29-11ea-9c43-19d601472e25.png)

Absent:
![image](https://user-images.githubusercontent.com/58787602/80593090-383c7580-8a29-11ea-8329-68631cef99e5.png)

## Testing Instructions

* Click any query
* notice the presence / absence of a blue dot
